src/components/OmouComponents/AddItemButton.js

```diff
@@ -14,11 +13,12 @@ const useStyles = makeStyles({
         cursor: 'pointer',
         height: height,
         width: width,
+        color: '#999999',
         background: '#F5F5F5',
         border: '1.5px dashed #999999',
         borderRadius: '5px',
         boxSizing: 'border-box',
-        boxShadow: '0 0.5rem 0.5rem rgba(0,0,0, 0.2)',
+        boxShadow: '0px 0px 8px rgb(196 196 196 / 60%)',
```

Comment:
set `'0px 0px 8px rgb(196 196 196 / 60%)'` as a variable in muiTheme.js put this under "theme colors". call it "defaultBoxShadow"